build.gradle

```diff
@@ -8,7 +8,7 @@ buildscript {
         classpath "org.jlleitschuh.gradle:ktlint-gradle:10.1.0"
     }
     ext {
-        compose_ui_version = '1.1.1'
+        compose_ui_version = '1.4.0-beta01'
```

Comment:
Why beta? Is there anything the beta gives you that the latest stable doesn't?

Reply:
the only reason why I was useing beta was because I thought an issue was occuring because of an issue with versioning, by the time I figured out why it was actually happening, everything was as high of a version it could go. changed to be a stable version.